build.gradle

```diff
@@ -407,24 +399,21 @@ ccd {
   configDir = file('build/definitions')
 }
 
-task bootWithCcd(type: org.springframework.boot.gradle.tasks.run.BootRun, dependsOn: [loadEnvSecrets, buildCCDXlsx], group: 'ccd tasks') {
-    classpath sourceSets.cftlib.runtimeClasspath
-    mainClass = 'uk.gov.hmcts.divorce.cftlib.RunWithCFT'
-    doFirst() {
-      // Required by Access Management for Java 17.
-      // https://github.com/x-stream/xstream/issues/101
-      ['java.lang', 'java.util', 'java.lang.reflect', 'java.text', 'java.awt.font'].each {
-        jvmArgs('--add-opens', "java.base/${it}=ALL-UNNAMED")
-      }
-      jvmArgs('--add-opens', 'java.desktop/java.awt.font=ALL-UNNAMED')
-      jvmArgs('-XX:ReservedCodeCacheSize=64m')
+check.dependsOn tasks.cftlibTest
 
-      project.file('./.aat-env').readLines().each() {
-        def index = it.indexOf("=")
-        def key = it.substring(0, index)
-        def value = it.substring(index + 1)
-        environment key, value
-      }
+bootWithCCD {
+    dependsOn(loadEnvSecrets, buildCCDXlsx)
+    environment 'IDAM_S2S-AUTH_URL', '${IDAM_S2S_URL}'
+    environment 'CCD_DOCUMENT_URL_PATTERN', '.*'
+    environment 'ROLE_ASSIGNMENT_S2S_AUTHORISED_SERVICES', 'ccd_gw,am_role_assignment_service,am_org_role_mapping_service,xui_webapp,aac_manage_case_assignment,ccd_data,nfdiv_case_api'
+    environment 'DATA_STORE_S2S_AUTHORISED_SERVICES', 'ccd_gw,fpl_case_service,ccd_data,ccd_ps,divorce_frontend,payment-api,xui_webapp,nfdiv_case_api,ccd_case_document_am_api,am_role_assignment_service'
```

Comment:
I wonder if we could move this from here to `RunWithCft` or `CftLibConfig` - just override these env vars for the services that need them rather than globally?

Reply:
Yes service specific properties will be possible in future. We can probably come up with a cleaner API for defining these things rather than having to set these service-specific magic properties.